.github/workflows/release-azure-pipelines.yml

```diff
@@ -59,6 +59,8 @@ extends:
         name: Azure-Pipelines-1ESPT-ExDShared
         image: windows-latest
         os: windows
+      roslyn:
+        copyLogsOnly: true

```

Comment:
I think it's worth adding context about how this section (and the explicit build below) prevents an unnecessary second build.

Reply:
Done!